Helper/PaymentMethods.php

```diff
@@ -316,7 +318,7 @@ protected function getCurrentCountryCode($store, $country = null)
             return $country;
         }
 
-        if ($country = $this->getQuote()->getBillingAddress()->getCountry()) {
+        if (!is_null($customerId) && $country = $this->getQuote()->getBillingAddress()->getCountry()) {
```

Comment:
If the customer is logged in, `$quote->getCustomerId()` will return an int. Otherwise, it returns `null`. That is why I am checking that `$customerId` is not `null`.